### ISSUE TITLE
Show divider with single line TextView 

### ISSUE THREAD

**Issue by Hiike** [google/flexbox-layout#384]
## Issues and steps to reproduce
When adding the app:showDivider="middle", app:flexWrap and dynamically adding TextView with setMaxLines(1) in FlexboxLayout. TextView's width is reduced 

![flexbox](https://user-images.githubusercontent.com/7040668/33325311-2db91986-d452-11e7-8a76-a2f81ab3f512.jpeg)

The extra space comes from the new line in the TextView which is not supposed to happen.

(Screenshot without setMaxLines(1))
![flexbox_2](https://user-images.githubusercontent.com/7040668/33325461-a3caa004-d452-11e7-92ae-087552d954cf.jpeg)

## Expected behavior
TextView showing the entire given String.

## Version of the flexbox library
0.3.1

## Link to code

In LinearLayout

    <com.google.android.flexbox.FlexboxLayout
        android:id="@+id/hobby_cell"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        app:flexWrap="wrap"
        app:showDivider="middle"
        app:dividerDrawable="@drawable/divider"
        android:layout_weight="1"
        />

    <Button
        android:id="@+id/edit_button"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:text="@string/button_edit"
        />

[code](https://gist.github.com/Hiike/b166d5f539ae1337894156aa0af38014)

**Comment by thagikura** [comment]
I tried to reproduce the issue, but wasn't able to reproduce it (https://github.com/google/flexbox-layout/pull/387)

Is it possible to have a look at your full code example instead of code snippets?
Some parameters are different from the code I tried to reproduce.

**Comment by Hiike** [comment]
Here's the [link](https://github.com/Hiike/FlexExample) to a repo that I just created that reproduce the issue

**Comment by thagikura** [comment]
Thanks for preparing the repo, I checked the code.

Looks like it's caused by the parent LinearLayout which has FlexboxLayout as its child.
Its orientation is set to `horizontal` and if the device width isn't large enough for having FlexboxLayout and the Button (Editer), the width of the FlexboxLayout is shrank to have both views.

When I deleted the Button, the texts inside the FlexboxLayout were showed entirely.
